x-pack/plugins/fleet/server/services/epm/packages/install.ts

```diff
@@ -1228,13 +1283,20 @@ export async function createInstallation(options: {
     ? true
     : undefined;
 
+  const manifestAsset = loadYmlFromAssetMap(assetsMap, `${pkgName}-${pkgVersion}/manifest.yml`);
```

Comment:
I think this is not needed either, the `humanReadableTitle` read from the manifest looks the same as `packageInfo.title`